### PR TITLE
Remove support of Rails 3.x

### PR DESCRIPTION
**Pull request by n-rodriguez**
I will rebase it on the master branch when https://github.com/appfolio/store_base_sti_class/pull/26 will be merged.

### CODE REVIEW

File: store_base_sti_class.gemspec [hunk 1]
```diff
@@ -20,9 +20,9 @@ Gem::Specification.new do |s|
   s.homepage = 'http://github.com/appfolio/store_base_sti_class'
   s.licenses = ['MIT']
   s.rubygems_version = '2.2.2'
-  s.summary = 'Modifies ActiveRecord 3.0.5 - 4.2.x with the ability to store the actual class (instead of the base class) in polymorhic _type columns when using STI'
+  s.summary = 'Modifies ActiveRecord 4.0.x - 5.0.x with the ability to store the actual class (instead of the base class) in polymorhic _type columns when using STI'
```
Comment (jonkessler) [inline store_base_sti_class.gemspec]: This shouldn't say 5.0.x since that's in the other pull request.

Comment (n-rodriguez) [inline store_base_sti_class.gemspec]: Sure!

**Comment by n-rodriguez** [comment]
> Awesome, thanks! I'll merge shortly.

Great :) Don't forget to merge [this one](https://github.com/appfolio/store_base_sti_class/pull/26) first, as we'll need to rebase.

**Comment by jonkessler** [comment]
@n-rodriguez Oh, I was thinking we would rebase the other one after this is merged. Seems easier that way.

**Comment by n-rodriguez** [comment]
As you want :)

**Pull request merged**